base_layer/wallet_ffi/tests/test_build_validation.py

```
@@ -0,0 +1,664 @@
+                }
+                print(f"❌ Environment check failed: {e}")
+            
+            # Build prerequisites
+            try:
+                # Check for basic build tools
+                prerequisites = {'rust': False, 'maturin': False}
+                
+                try:
+                    subprocess.run(['rustc', '--version'], capture_output=True, timeout=5)
+                    prerequisites['rust'] = True
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)